tests/system/providers/amazon/aws/example_eks_with_nodegroup_in_one_step.py

```diff
@@ -116,13 +116,15 @@ def delete_launch_template(template_name: str):
 
     describe_pod = BashOperator(
         task_id="describe_pod",
-        bash_command=""
-        # using reinstall option so that it doesn't fail if already present
-        "install_aws.sh --reinstall " "&& install_kubectl.sh --reinstall "
-        # configure kubectl to hit the cluster created
-        f"&& aws eks update-kubeconfig --name {cluster_name} "
-        # once all this setup is done, actually describe the pod
-        "&& kubectl describe pod {{ ti.xcom_pull(key='pod_name', task_ids='run_pod') }}",
+        bash_command=f"""
+            install_aws.sh || true;
+            install_kubectl.sh || true;
+            # configure kubectl to hit the cluster created
+            aws eks update-kubeconfig --name {cluster_name};
+            # once all this setup is done, actually describe the pod
+            echo "vvv pod description below vvv";
+            kubectl describe pod {{{{ ti.xcom_pull(key='pod_name', task_ids='run_pod') }}}};
+            echo "^^^ pod description above ^^^" """,
```

Comment:
I know we try keep the sys tests DAMP but having this script copy/pasted so many times makes me sad :crying_cat_face:

Reply:
I was going to say the same thing.  I know why he/we did it and don't necessarily want it changed, but it made me a sad panda reading it LOL

Reply:
One of our early decisions was that every test should be fully self-contained as much as possible.  In that spirit, thus makes the most sense.